pkg/fw/Dockerfile

```diff
@@ -37,6 +37,10 @@ RUN ln -s brcmfmac43455-sdio.raspberrypi,3-model-b-plus.txt /lib/firmware/brcm/b
     ln -s brcmfmac43430-sdio.raspberrypi,3-model-b.txt /lib/firmware/brcm/brcmfmac43430-sdio.txt
 # symlinks for Visionfive1 riscv64 boards
 RUN ln -s ../cypress/cyfmac43430-sdio.bin /lib/firmware/brcm/brcmfmac43430-sdio.starfive,visionfive-v1.bin
+# symlinks for kondor wifi device (brcmfmac4356)
```

Comment:
is this device called kondor or misspelled of Kontron?

Reply:
This seems like a typo given the symlink below being called Kontron.

But that raises the question whether we need a different symlink if some other server uses the same WiFi hardware. Do you know @jsfakian ?

Reply:
This device is called Kontron. I was surprised as well. :)

Reply:
@eriknordmark I do not know if we need a different symlink for the WiFi device brcmfmac4356 on other machines, but I think it could often be the case.